eng/common/scripts/ICommon.ps1

```diff
@@ -0,0 +1,27 @@
+$global:RepoRoot = Resolve-Path "${PSScriptRoot}..\..\..\.."
```

Comment:
I would just call this script "common.ps1" putting `I` in front is supposed to represent an interface but I don't think that works well for script names.